src/client/driver/driver.js

```diff
@@ -415,10 +415,13 @@ export default class Driver {
     }
 
     _onCustomCommand (command) {
-        this.customCommandHandlers[command.type](command).then(response => {
+        var handler = this.customCommandHandlers[command.type];
+
+        handler(command).then(result => {
             this._onReady(new DriverStatus({
```

Comment:
If you place `new DriverStatus({isCommandResult: true,, result})` in one line linter will not detect wrong indent after pressing `ctrl+alt+L` :)